cmake/evt-core_compiler.cmake

```diff
@@ -102,4 +105,4 @@ endif()
 
 set(CMAKE_EXE_LINKER_FLAGS  "${CMAKE_EXE_LINKER_FLAGS} -mfloat-abi=hard \
                             -specs=nano.specs -specs=nosys.specs \
-                            -lc -lm -lnosys -Wl,--gc-section")
+                            -lc -lm -lnosys -Wl,--gc-section -Wl,--print-memory-usage")
```

Comment:
As discussed in this issue: https://rit-evt.atlassian.net/browse/FWT-64, the `gc-section` flag is important for reducing linking of unnecessary objects. Why have you removed it here?

Reply:
Where do you see it being removed? I see `-Wl,--gc-section -Wl,--print-memory-usage` which adds another flag, nothing is removed

Reply:
Well, I'm either blind or stupid. Although, I believe this could be simplified to 
```suggestion
                            -lc -lm -lnosys -Wl,--gc-section,--print-memory-usage")
```